packages/pangea-sdk/pangea/verify_audit.py

```diff
@@ -99,6 +99,37 @@ def _verify_hash(data: t.Dict, data_hash: str) -> t.Optional[bool]:
     return succeeded
 
 
+def _verify_unpublished_membership_proof(
+    root_hash, node_hash: str, proof: t.Optional[str]
+) -> t.Optional[bool]:
+    global pub_roots
+
+    log_section("Checking unpublished membership proof")
+
+    if proof is None:
+        succeeded = None
+        logger.debug("Proof not found")
+    else:
+        try:
+            logger.debug("Decoding hashes")
+            root_hash_dec = decode_hash(root_hash)
+            node_hash_dec = decode_hash(node_hash)
+
+            logger.debug("Calculating the proof")
+            proof_dec = decode_membership_proof(proof)
```

Comment:
are we sure that this does not crash when proof = ""?

Reply:
yes